stories/Common/Modal.stories.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
```

Comment:
Would you move this component to `client/components/Common/Modal.stories.ts`?
I'd like to place story files at the same level as components.

Reply:
I sympathize your idea.
So I move a story file to the same level as components.